spark/util/helper_functions/misc_functions.py

```diff
@@ -87,3 +87,27 @@ def helper_random(self, *args):
 @ignite_global
 def helper_randint(self, *args):
     return random.randint(0, args[0])
+
+
+@validate_args([Real, Real, Real, Real])
+@ignite_global
+def helper_bounding_box(self, *args):
+    return args
+
+@validate_args([list, list], [list, list, bool], [tuple, tuple], [tuple, tuple, bool])
+@ignite_global
+def helper_collided(self, *args):
+    x1, y1, width1, height1 = args[0]
+    x2, y2, width2, height2 = args[1]
+    overlap_on_equal = len(args) == 3 and args[2]
+    return self.axis_overlapped(x1, width1, x2, width2, overlap_on_equal) and self.axis_overlapped(y1, height1, y2, height2, overlap_on_equal)
+
+@validate_args([Real, Real, Real, Real], [Real, Real, Real, Real, bool])
+@ignite_global
+def helper_axis_overlapped(self, *args):

```

Comment:
I would suggest using a keyword argument for the overlap_on_equal
```python
def helper_axis_overlapped(self, *args, overlap_on_equal=False)
```

Reply:
But I'm also fine with it as is

Reply:
Forgot to implement this; I'll get this done in a future revision since it's a very unused feature rn